solutions/secure-regional-bucket/variables.tf

```diff
@@ -48,6 +48,13 @@ variable "key_name" {
   description = "The name for the new key for the Object Storage bucket. Does not apply if a key is specified in `existing_kms_key_crn`."
 }
 
+variable "ibmcloud_kms_api_key" {
+  type        = string
+  description = "The IBM Cloud API key that can create a root key and key ring in the key management service (KMS) instance. If not specified, the 'ibmcloud_api_key' variable is used. Specify this key if the instance in `existing_kms_instance_crn` is in an account that's different from the Event Notifications instance. Leave this input empty if the same account owns both instances."
```

Comment:
The first sentence should reference the COS to KMS authorisation policy, also created with this key if is not null.

It is a DA, so we should probably get @SirSpidey to review the wording.

`The IBM Cloud API Key that is used to create an authorization policy, key ring and root key for an existing KMS instance. Specify this key if the instance in `existing_kms_instance_crn` is in an account that is different from the COS bucket. Leave this input empty if the same account owns the KMS instance and  the COS bucket.`

My interpretations is that there is no Event Notifications instance, so that sentence should not be used. The final sentence does not make sense because it is a KMS instance and bucket.

Reply:
Keira's description is pretty standard for that variable, with good capitalization. Update as needed for this solution. Justmake sure: 
- that names of variables are enclosed in backticks \`, not single quotation marks `'` (`ibmcloud_api_key`). 
- that you don't use **COS** or **Cloud Object Storage** in any descriptions. Use either **Object Storage** or **IBM Cloud Object Storage**.
- to use the phrase, `Not used if...` instead of "Leave this input empty..."

Reply:
Thank you. Yes, I have been making this same change in a few different places. I will update this to match this module.